packages/compass-explain-plan/src/components/explain-tree/tree-layout.tsx

```diff
@@ -128,6 +157,27 @@ function TreeLayout<T>({
           height={height}
           style={{ position: 'absolute', top: 0, left: 0 }}
         >
+          <defs>
+            <marker
+              markerWidth="4"
+              markerHeight="4"
+              refX="7.5"
+              refY="7.5"
+              viewBox="0 0 15 15"
+              orient="auto-start-reverse"
+              id="arrowhead"
+            >
+              <polyline
+                points="0,7.5 7.5,3.75 0,0"
+                fill="none"
+                strokeWidth="3"
+                stroke={arrowColor}
+                strokeLinecap="round"
+                transform="matrix(1,0,0,1,2.5,3.75)"
+                strokeLinejoin="round"
+              />
```

Comment:
This seems to be off from designs, in the designs the arrow "wings" are less narrow:

![image](https://github.com/mongodb-js/compass/assets/5036933/46a29a94-2271-41e6-af9b-719076b433ee)

vs

![image](https://github.com/mongodb-js/compass/assets/5036933/b5ab5536-5276-48f9-a519-33e3abb5c69e)